lib/awful/spawn.lua

```diff
@@ -323,14 +323,12 @@ end
 --- Asynchronously spawn a program and capture its output.
 -- (wraps `spawn.with_line_callback`).
 -- @tparam string|table cmd The command.
--- @function callback Function with the following arguments:
+-- @tab callback Function with the following arguments
```

Comment:
> Also, you reverted what I believed were necessary corrections.

Do you think this still after the discussion?

Should we rather use `@tparam table` for explicitness?